core/src/utils/animation/animation.ts

```diff
@@ -979,8 +860,6 @@ export const createAnimation = (animationId?: string): Animation => {
     if (supportsWebAnimations) {
       setAnimationStep(0);
       updateWebAnimation();
-    } else {
```

Comment:
Can we add some comments where we still use the `supportsWebAnimations` checks (or even a comment where the const is defined) that explains why we still use this support check?